editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx

```diff
@@ -670,129 +679,16 @@ describe('Select Mode Double Clicking With Fragments', () => {
     checkSelectedPaths(renderResult, [desiredPaths[1]])
   })
 
-  describe('With Code in navigator FS on', () => {
-    setFeatureForBrowserTests('Code in navigator', true)
-    it('Single click and three double clicks will focus a generated Card', async () => {
-      // prettier-ignore
-      const desiredPaths = createConsecutivePaths(
-      'sb' +                 // Skipped as it's the storyboard
-      '/scene-CardList' +    // Skipped because we skip over Scenes
-      '/CardList-instance' + // Skipped because we skip component children of Scenes
-      ':CardList-Root' +     // Skipped because we skip over root elements
-      '/CardList-Col',      // <- Single click
-      '/6d6',               // <- First double click selects the expression item
-      '/CardList-Card~~~1', // <- Second *and* third double click, as the Second is required to focus it
-    )
-
-      const renderResult = await renderTestEditorWithCode(
-        TestProjectAlpineClimb,
```

Comment:
was it deliberate to change the test project from TestProjectAlpineClimb to TestProjectAlpineClimbWithFragments? am I reading the diff wrong?

Reply:
I think the diff is hard to follow, there are two versions of these tests, both with `TestProjectAlpineClimb` and `TestProjectAlpineClimbWithFragments` (just like earlier)